README.md

```diff
@@ -15,3 +15,4 @@ Some of the plugins use additional modules, including:
   - [lua-http](https://github.com/daurnimator/lua-http)
   - [dkjson](http://dkolf.de/src/dkjson-lua.fsl/home)
   - [lua-gumbo](https://github.com/craigbarnes/lua-gumbo)
+  - [utf8](https://luarocks.org/modules/dannote/utf8)
```

Comment:
You don't need a library; there is a `utf8` library built into lua 5.3. www.lua.org/manual/5.3/manual.html#pdf-utf8.char

To get it in other lua versions you should depend on [compat-5.3](https://github.com/keplerproject/lua-compat-5.3)